src/BasicRum/DiagramSchema/Filters/Primary/DeviceType.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\BasicRum\DiagramSchema\Filters\Primary;
+
+class DeviceType implements \App\BasicRum\DiagramSchema\Filters\FilterableInterface
+{
+    public function getSchema(): array
+    {
+        $schema = [
+            'device_type' => [
+                'type' => 'object',
+                'properties' => [
+                    'search_value' => [
+                        'enum' => [1, 2, 3, 4, 5],
+                        'type' => 'integer',
+                    ],
+                    'condition' => [
+                        'enum' => ['is'],
```

Comment:
Hello @korzol 

Do you think that the list of values here may change in the future? I think that we need to keep in mind this and fetch the values dynamically.

Reply:
Hello @ceckoslab 
Kindly explain here 
> fetch the values dynamically

What would be / is the source here?

Thanks

Reply:
Hello @korzol 

In regards of **condition** in the code I see **'enum' => ['is']** but what if we want to filter to create a diagram that shows everything but not a specific device. E.g. I want to have a diagram that shows everything that is not a mobile device?

However I think that we need to move to another task soon and come back again to the Dynamic JSON when we define a bit better the architecture and how the system components are connected to each other.

Could you please skip this the further work in DeviceType.php ?

Reply:
I absolutely agree with the following:
> In regards of condition in the code I see 'enum' => ['is'] but what if we want to filter to create a diagram that shows everything but not a specific device. E.g. I want to have a diagram that shows everything that is not a mobile device?

That's what my question was about.  We need to add more conditions there however, once you think it should be dynamic and I agree with that - we need to think what would be the source for conditions. Even if not right now, later we still have to find a solution for this.